eclipse-scout-core/src/util/objects.ts

```diff
@@ -368,14 +368,32 @@ export const objects = {
   },
 
   /**
-   * @returns true if the given object is an object: no primitive type (number, string, boolean, bigint, symbol), no array, not null and not undefined.
+   * @deprecated The method was renamed to {@link isObject}. Use the new name or consider using {@link isPOJO} instead.
```

Comment:
If we deprecate this method, we should also migrate all code to the new function. Otherwise lots of deprecated warnings will popup. Maybe only consider this deprecation for 25.1 and keep the existing 24.2 API stable?

Reply:
Yes, we should migrate existing code. I would rather remove this method with 25.1. completely

Reply:
I'll add the new function on 25.1 only and use $.isPlainObject now